### ISSUE TITLE
error while compiling in android studio

### ISSUE THREAD

**Issue by patelbhavin8008** [redsolution/xabber-android#258]
Hi,

I tried to complie the source with android studio. I just downloaded source code and imported project in android studio and with out doing any changes or modification i just tried to do complie but i get below errors in log file.

============= Log File of Android Studio================
5:46:37 PM Update Property Files
           The structure of following Android modules was changed:
           xabber-android-master
           Would you like to update related project.properties files?
           Only once
           Always for these modules
           Never for these modules
5:59:45 PM Deprecated make implementation
           Old implementation of "Make" feature is enabled for this project.
           It has been deprecated and will be removed soon.
           Please enable newer 'external build' feature in Settings | Compiler.
6:00:07 PM Compilation completed with 3 errors and 100 warnings in 22 sec
6:00:35 PM Deprecated make implementation
           Old implementation of "Make" feature is enabled for this project.
           It has been deprecated and will be removed soon.
           Please enable newer 'external build' feature in Settings | Compiler.
6:00:41 PM Compilation completed with 3 errors and 10 warnings in 6 sec
6:01:26 PM Compilation completed with 3 errors and 103 warnings in 23 sec
==================================================

**Comment by andrewnenakhov** [comment]
@grigoryfedorov close it pls if it is no longer relevant.

**Comment by grigoryfedorov** [comment]
Moved to Android studio/Gradle build.
